src/main/java/uk/gov/ons/ssdc/supporttool/model/dto/messaging/EventTypeDTO.java

```diff
@@ -4,5 +4,6 @@
   DEACTIVATE_UAC,
   ADDRESS_NOT_VALID,
   REFUSAL_RECEIVED,
-  FULFILMENT
+  FULFILMENT,
+  UPDATE_SAMPLE_SENSITIVE
```

Comment:
This will need to be added to Case Processor, Case API and ROps too

Reply:
tis there already